fix(example): with-cloudflare-workers example (#5585)
supabase/supabase: examples/with-cloudflare-workers/README.md

# Supabase with Cloudflare Workers

> `supabase-js` uses the `cross-fetch` library to make HTTP requests, but that does not work pn Cloudflare Workers.

Reviewer: ```suggestion
`supabase-js` uses the `cross-fetch` library to make HTTP requests, which has issues on Cloudflare Workers.
```

> We're asking Supabase Client to use Cloudflare Workers' in-built `fetch` method to make HTTP requests using the `fetch` option

Reviewer: ```suggestion
To solve this we can use Cloudflare Workers' in-built `fetch` method to make HTTP requests using the optional `fetch` parameter in the `supabase-js` client.
```
